sentry-core/src/backtrace_support.rs

```diff
@@ -159,6 +157,10 @@ fn parse_crate_name(func_name: &str) -> Option<String> {
         .map(|cr| cr.as_str().into())
 }
 
+/// Processes a stacktrace.
```

Comment:
Same below. Either:
```suggestion
/// Processes a `Stacktrace`.
```

Or:
```suggestion
/// Processes a stack trace.
```